src/SD.h

```diff
@@ -70,6 +70,9 @@ class SDClass {
   boolean begin(uint8_t csPin = SD_CHIP_SELECT_PIN);
   boolean begin(uint32_t clock, uint8_t csPin);
   
+  //call this when a card is removed. It will allow you to inster and initialise a new card.
```

Comment:
SHould be insert and not inster

Reply:
Sure. But this is the text from the original patch. The typo is like a fingerprint, so you know where it came from.

Reply:
I disagree. If it's desirable to document the source of the code then just add a comment instead of relying on a minor typo that may get fixed in the Adafruit library.

Reply:
Whatever. I've fixed the typo